drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -101,10 +101,7 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
     }
 
     /* get an 8-byte unique ID to use as hardware address */
-    luid_get(addr_long.uint8, IEEE802154_LONG_ADDRESS_LEN);
-    /* make sure we mark the address as non-multicast and not globally unique */
-    addr_long.uint8[0] &= ~(0x01);
-    addr_long.uint8[0] |=  (0x02);
+    luid_get_eui64(addr_long);
```

Comment:
Unrelated change

Reply:
Sorry

note to me: "never work in master"